CONTRIBUTING.md

```diff
@@ -321,3 +321,38 @@ stripped with no prior warning or consent from the member in question.
 Earning this privilege should be considered to be no small feat and is by no
 means guaranteed by any quantifiable metric. It is a symbol of great trust of
 the community of this project.
+
+## Point Release Procedure
+
+*   3 Stages
+    *   **Strategy Discovery**
+        *   Develop long term priorities, strategy and roadmap for the SDK
+        *   Release committee not yet defined as there is already a roadmap that can be used for the time being
+    *   **Concept Approval**
+        *   Time bound period for Request for Comment (RFC) on Architectural Decision Records (ADR)
+        *   Release committee criteria
```

Comment:
I think of "release committee" as usually in the context of a specific release (`vX.X.X`). Can we call it something else here? ADR Approval committee?

Reply:
Exactly. A stable release series is generally identified by `maj.min` (e.g. `0.39`, `0.40`) and should have *release managers* (or committee, or whatever). All those release series `X.Y` that are meant to be maintained for longer than just until `X.Y+1` is out **should** have a *stable release policy*.